buildSrc/src/main/groovy/java-common-conventions.gradle

```diff
@@ -14,9 +14,7 @@ if (!isCiServer) {
     tasks.withType(JavaExec).configureEach {
         systemProperty 'spring.profiles.include', 'human-readable-logging'
     }
-    tasks.withType(Test).configureEach {
-        systemProperty 'spring.profiles.include', 'human-readable-logging'
-    }
+    tasks.withType(Test).configureEach {}
```

Comment:
I can take this out if others feel strongly, but I think its presence is a way of saying "this is how you configure tests at the top level" and "there is no configuration right now"